WowPacketParser/Parsing/Parsers/BattlegroundHandler.cs

```diff
@@ -1030,5 +1030,14 @@ public static void HandleArenaTeamInviteServer(Packet packet)
             packet.ReadCString("Player Name");
             packet.ReadCString("Arena Team Name");
         }
+
+        [Parser(Opcode.SMSG_REPORT_PVP_PLAYER_AFK_RESULT)]
+        public static void HandleReportPvPPlayerAfkResult(Packet packet)
+        {
+            packet.ReadPackedGuid128("Offender");
+            packet.ReadByte("Result");
+            packet.ReadByte("NumBlackMarksOnOffender");
+            packet.ReadByte("NumPlayersIHaveReported");
```

Comment:
wrong order

Reply:
it's wrong in TC lel, check sniffs

Reply:
_worldPacket << Offender;

    _worldPacket << NumPlayersIHaveReported;
    _worldPacket << NumBlackMarksOnOffender;
    _worldPacket << Result;

this order, but wouldnt try to find tihs packet sniffed atm, it'll take too lot time

Reply:
you are wrong @soulfrost i checked handlers of 720, 735 and 801, structure is correct.

Reply:
oh god, thank u for report, seems like i had checked it tooo long time ago